datanode/gateway/graphql/schema.graphql

```diff
@@ -6111,3 +6156,19 @@ type ReferrerRewardsGenerated {
   "The amount of rewards generated per party"
   generatedReward: [PartyAmount!]!
 }
+
+"Volume discount program information"
+type VolumeDiscountProgram {
+  "Unique ID generated from the proposal that created this program."
+  id: ID!
+  "Incremental version of the program. It is incremented each time the volume discount program is edited."
+  version: Int!
+  "Defined tiers in increasing order. First element will give Tier 1, second element will give Tier 2, etc."
+  benefitTiers: [VolumeBenefitTier!]!
+  "Timestamp as RFC3339Nano, after which when the current epoch ends, the programs status will become STATE_CLOSED and benefits will be disabled."
```

Comment:
Is there a state_closed? Same question as on protos, based on this comment in a referrals API PR https://github.com/vegaprotocol/vega/pull/9550#discussion_r1335859626